.github/workflows/remove-label-on-approve.yml

```
@@ -1,12 +1,12 @@
-name: Detect Pull Request Review
+name: Approval
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)